_datasets/sample-dataset.md

```diff
@@ -18,4 +18,5 @@ category:
   - Education
 maintainer: Tim Wisniewski
 maintainer_email: tim@timwis.com
+comments: true
```

Comment:
Yes, it should be a possibility.

Matteo

> Il giorno 08 mag 2016, alle ore 22:42, Patrick Connolly <notifications@github.com> ha scritto:
> 
> In _datasets/sample-dataset.md <https://github.com/timwis/jkan/pull/117#discussion_r62438103>:
> 
> > @@ -18,4 +18,5 @@ category:
> >    - Education
> >  maintainer: Tim Wisniewski
> >  maintainer_email: tim@timwis.com
> > +comments: true
> Would it not make more sense to default to true in _config.yml?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/timwis/jkan/pull/117/files/b234c379bdc352334d4107b4adaadf70797ba727#r62438103>